src/lib/Microsoft.Health.Fhir.Ingest/Service/EventProcessingMeter.cs

```diff
@@ -0,0 +1,36 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.EventHubs;
+
+namespace Microsoft.Health.Fhir.Ingest.Service
+{
+    public class EventProcessingMeter : IEventProcessingMeter
+    {
+        public Task<EventStats> CalculateEventStats(IEnumerable<EventData> events)
+        {
+            double ingressSizeBytes = 0;
+
+            foreach (var e in events)
+            {
+                var bodySizeBytes = e.Body.Array.Length;
+                ingressSizeBytes = ingressSizeBytes + bodySizeBytes;
+                ingressSizeBytes = e.Properties.Aggregate(ingressSizeBytes, (current, entry) => current + Encoding.UTF8.GetByteCount(entry.Key) + Encoding.UTF8.GetByteCount(entry.Value.ToString()));
```

Comment:
>Aggregate [](start = 48, length = 9)

Looks like a forgot to publish this comment.  Suggestion: refactor into a common function for dictionaries.